apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -146,19 +153,34 @@ export default class AnimationPickerBody extends React.Component {
     if (this.props.hideBackgrounds) {
       categories = categories.filter(category => category !== 'backgrounds');
     }
+    // Level-specific animations currently have the following categories:
+    // "level_animations", "backgrounds", "animals".
+    // We want to hide the "animals" category which has only one animation in it.
+    // It will be displayed in the "all" category added below.
+    // There is an 'animals' category because a level-specific animation was uploaded with the
+    // 'animals' category BEFORE the sprite upload feature dictated that level-specific animations
+    // can only be categorized as "costume" or "background".
+    // We also replace 'level_animations' with the 'costumes' category.
+    // This is a more accurate label for level-specific animations that are not backgrounds.
+    if (categories.includes('level_animations')) {
+      categories = categories.filter(category => category === 'backgrounds');
```

Comment:
With https://github.com/code-dot-org/code-dot-org/pull/55333/commits/c3d75cf8babd8d633f4b112f04d2f1e706407814 - I no longer need to update the name 😄 . I changed filter to making sure only categories 'level_costumes' and 'backgrounds' are included just in case someone manually adds a level-specific category that is not either 'backgrounds' or ''. The animation will show up in the 'all' category.